lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -1077,6 +1082,43 @@ def build_accessible_object_scope(associations, options = {})
       end
     end
 
+    def accessible_objects_for_operations_function(user_id, operations, options)
+      prohibition_names = []
+
+      operation_names = operations.map do |o|
+        name = operation_identifier(o)
+        # Store prohition name if not ignoring prohibitions
+        prohibition_names << prohibition_identifier(name) unless options[:ignore_prohibitions]
+        name
+      end
+
+      all_names = operation_names + prohibition_names
+      accessible_map = all_names.map { |o| [o, []] }.to_h
```

Comment:
```suggestion
      accessible_map = all_names.index_with([])
```
part of activesupport but this gem has activesupport as a dependency so we should be covered

Reply:
also, does this block mean that the function won't return the operation name key if there are no accessible object uris?

Reply:
if so, would it be easy to change the function to just return an empty string when there are no accessible object uris? not a big deal but that could simplify the code here a bit.

Reply:
That shape doesn't seem to be inline with the existing specs to me.